lib/python/bind_operators.h

```diff
@@ -83,6 +83,90 @@ void bind_astype(py::class_<T, Ignored...> &c) {
         :rtype: Union[scipp.Variable, scipp.DataArray])");
 }
 
+template <class T, class... Ignored>
+void bind_to(py::class_<T, Ignored...> &c) {
+  c.def(
+      "to",
+      [](const T &self, const py::object &dt,
```

Comment:
Since this is implemented in C++ now, it would be nice to have it in the `variable` package so it is accessible in C++. (Only the stuff below releasing the GIL, of course.)

Reply:
I actually recommended above to move it to Python, to avoid a number of potential pitfalls.